cli/oh-my-zsh/README.md

```diff
@@ -1,4 +1,4 @@
 ## How to install
 
-- copy the ```docsearch``` folder in ```~/.oh-my-zsh/custom/plugins/```
-- enable the plugin in your ```zshrc``` by modify the plugins variable: ``plugins=([...], docsearch)
\ No newline at end of file
+- copy the `docsearch` folder in `~/.oh-my-zsh/custom/plugins/`
```

Comment:
You should add a paragraph about what this plugin does. I didn't even know we had one, and I still don't really know what it does. Is that autocompletion?

Reply:
I do not recall cc @maxiloc

Reply:
If it's not used, let's remove it. But if it's useful, let's use it :)